src/index.ts

```diff
@@ -99,7 +115,8 @@ function clustergrammer_gl(
         },
       },
       utils: {
-        highlight: updateSearchedRows(regl, store, catArgsManager, camerasManager),
+        highlightTriangles: updateSearchedRows(regl, store, catArgsManager, camerasManager),
+        hightlightRowsCols: updateSearchedHightlights(regl, store, catArgsManager, camerasManager),
```

Comment:
Since we are changing the API of our returned object instead of just adding to it, this should probably cause a minor version change, correct?

Reply:
Yep!

Reply:
Could you bump the version in `package.json` as part of this PR then, please?